grouper/fe/templates/service.html

```diff
@@ -238,8 +238,15 @@ <h4 class="modal-title">Remove User</h4>
     {% endif %}
 {% endmacro %}
 
-<div class="modal fade" id="auditModal" tabindex="-1" role="dialog"
-      aria-labelledby="auditModal" aria-hidden="true">
+<div class="modal fade"
```

Comment:
Unneeded churn?

Reply:
The tag had to change due to the introduction of the UJS data-* fields anyway